template/add-users-group.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="./public/styles.css" />
+    <title>PostIt - My Groups</title>
+  </head>
+  <body>
+    <nav class="navbar navbar-dark bg-dark">
+        <h2 class="navbar-text">PostIt - Add Users to a Group</h2>
+      </nav>
+    <section class="vh-100" style="background-color: #508bfc">
+      <div class="container py-5 h-100">
+        <div class="row d-flex justify-content-center align-items-center h-100">
+          <div class="col-12 col-md-8 col-lg-6 col-xl-5">
+            <div class="card shadow-2-strong" style="border-radius: 1rem">
+              <div class="card-body p-5 text-left">
+                <div class="mb3">
+                    <form method="POST">
+                      <label class="form-label" for="group">Add users to a group</label>
+                      <select name="group" class="form-select" id="group">
+                        <option selected>Select the group</option>
+                        >
+                        <option>Avengers</option>
+                        <option>Kibala Boys</option>
+                        <option>Family</option>
+                      </select>
+              
+                      <label class="form-label" for="addUsers">Select users to add to group</label>
+                      <div class="select-users">
+                        <div>
+                          <input type="checkbox" class="form-check" value="1" /><img
```

Comment:
Add an id to the input to match the `for` attribute on the label
```suggestion
                          <input type="checkbox" class="form-check" value="1" id="addUsers" /><img
```